dataPrepII_PCC/makePCCRateFile.py

```diff
@@ -154,7 +154,7 @@ def fillClusterCounts(pcc_tree, tree_scan_info, inData, PCC_BCID):
                   (100. * i / num_entries, time.strftime("%H:%M:%S", eta))
         if data.scanIndex < 0:
             print "WARNING Found scanIndex < 0. Should this happen?"
-        num_clus = ClusterCount(data)
+        num_clus = GetCount(data,DataType)
```

Comment:
If the suggestion above concerning rates per layer is ok, add

PixelLayer as an argument